src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,16 +4,22 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class.getName());
+
     @Override
     public Order completeOrder(Long userId) {
         // TODO: add log message about method completeOrder was called
+        logger.trace("Method 'completeOrder' was called");
```

Comment:
same, lets add info about incoming parameters

Reply:
what about this comment?

Reply:
Please excuse me for my carelessness